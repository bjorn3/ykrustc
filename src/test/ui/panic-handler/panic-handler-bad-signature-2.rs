// compile-flags:-C panic=abort

#![no_std]
#![no_main]
#![feature(lang_items)]

use core::panic::PanicInfo;

#[panic_handler]
fn panic(
    info: &'static PanicInfo, //~ ERROR argument should be `&PanicInfo`
) -> !
{
    loop {}
}

#[lang = "yk_swt_rec_loc"]
fn yk_swt_rec_loc(_crate_hash: u64, _def_idx: u32, _bb: u32) {}
