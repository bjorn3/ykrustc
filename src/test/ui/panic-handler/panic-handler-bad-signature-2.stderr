error: argument should be `&PanicInfo`
  --> $DIR/panic-handler-bad-signature-2.rs:11:11
   |
LL |     info: &'static PanicInfo, //~ ERROR argument should be `&PanicInfo`
   |           ^^^^^^^^^^^^^^^^^^

error: aborting due to previous error

